public/components/navigation/navigation.stache

```diff
@@ -3,22 +3,22 @@
 		<a href="/" class="pull-left main-logo">BitBalls</a>
 		<ul class="main-menu nav navbar-nav">
 		  <li role="presentation" {{#routeCurrent page='tournaments'}}class='active'{{/routeCurrent}}>
-				<a href="{{routeUrl page='tournaments'}}">Tournaments</a>
+				<a href="{{routeUrl (page='tournaments')}}">Tournaments</a>
```

Comment:
Oh, if it’s a helper then yes, it makes more sense to leave the space. The [docs](http://canjs.github.io/canjs/doc/can-stache.expressions.html) say that helpers can be written as `{{method hash=key}}`, so I’m curious why that didn’t work or was changed here.